package.xml

```diff
@@ -29,7 +29,7 @@
 -->
 
 <package format="2">
-  <name>ceres-solver</name>
+  <name>ceres_solver</name>
```

Comment:
This is a good workaround for now, but just for posterity I opened this to try and avoid the need for it: https://github.com/ament/ament_package/pull/49

But I haven't had time to iterate on that.